starter-code/css/main.css

```diff
@@ -1,8 +1,185 @@
-/*
-Colors:
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
 
-*/
+body {
+    font-family: "roboto", 'Open Sans';
+    width: 100%;
+}
+
+
+/* Header */
+
+.header {
+    position: fixed;
+    margin: auto 20px;
+    height: 60px;
+    width: 100%;
+}
+
+.logo {
+    width: 200px;
+    height: 60px;
+    float: left;
+}
+
+.header ul {
+    display: inline-block;
+    margin-right: 50px;
+    float: right;
+    right: 20px;
+    line-height: 60px;
+}
+
+.header li {
+    display: inline-block;
+    text-decoration: none;
+    color: #1A1A1A;
+    margin: 0 10px ;
+    list-style-type: none;
+    font-size: 18px;
+}
+
+
+/* Main box */
+
+
+.mainbox {
+    height: 1000px;
+    width: 100%;
+    background-image: url("../images/landing.jpg");
+    background-size: cover;
+    color: #FFF;
+    position: relative;
+    float: top;
+    top: 100px;
+}
+
+.text {
+    position: relative;
+    width: 50%;
+    height: 30%;
+    left: 25%;
+    top: 30%;
+}
+
+
+.mainbox h1 {
+    font-size: 70px;
+    display: flex;
+    justify-content: center;
+}
```

Comment:
I think `text-align` is better here. You should add the `display:flex` property to containers with items in it that you want to organize with more of a row/column logic.  If its just text you can use text-align :)